src/cli/cmdCli.ts

```diff
@@ -165,6 +168,33 @@ export class CmdCli implements Cli {
       // });
     });
   }
+
+  async executeExec(cmd: CliCommand, opts: ExecOptions = {}): Promise<CliExitData> {
+    if (cmd.cliCommand.startsWith('func')) {
+      const toolLocation: string = CmdCliConfig.tools.func.location;
+      if (toolLocation) {
+        // eslint-disable-next-line no-param-reassign
+        cmd.cliCommand = toolLocation
```

Comment:
i think you can remove the ternary operator as you are already checking toolLocation in the `if` above

Reply:
done